src/components/challengeWidget/challengeWidget.css

```diff
@@ -1,5 +1,6 @@
 .challengeWidget {
   width: 100%;
+  height: 50%;
```

Comment:
I'm not sure it's a good idea to control the height like this. What were trying to achieve here?